ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml

```diff
@@ -23,6 +23,8 @@ promotion:
 raw_steps:
 - pipeline_image_cache_step:
     commands: |
+      #NOTE(elfosardo): use ironic repos, cause we're special
+      curl http://base-4-11-rhel-8-server-ironic.ocp.svc > /etc/yum.repos.d/base-4-11-rhel-8-server-ironic.repo
```

Comment:
This URL seems unusual (and I don't think it I can test it locally) so just making sure it's not missing anything - is the repofile coming off a "magic" OCP service that's hosting it? If this is intended than LGTM.

Reply:
this is a "repository service" created by the release controller
the real repo configuration is stored in https://github.com/openshift/release/blob/master/core-services/release-controller/_repos/ocp-4.11-rhel-8-server-ironic.repo
the release controller translates that here https://github.com/openshift/release/blob/master/clusters/build-clusters/common/release-controller/rpms-ocp-4.11.yaml#L489

Reply:
Thank you for your explanation Riccardo, I anticipated it's something along these lines. LGTM.